package.json

```diff
@@ -64,19 +65,20 @@
     "jest": "^23.6.0",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.5.0",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
+    "react": "^16.9.0",
```

Comment:
React Hooks support started in 16.8 and also we might as well keep up (bigmaven is already using hooks for example!)